NEXT_CHANGES.rst

```diff
@@ -11,4 +11,3 @@ links to relevant Issues, Discussions, and PR's on github with the following for
 `Issue #XX <https://github.com/ixdat/ixdat/issues/XX>`_
 
 `PR #XX <https://github.com/ixdat/ixdat/pulls/XX>`_
-
```

Comment:
Can you describe the bug that this PR fixes?
Something like "fixed error that came up when trying to reuse axes for MSPlotter if both left and right axes were in use"... but I'm not sure.